custom_components/unifiprotect/views.py

```diff
@@ -32,9 +33,10 @@ def _get_access_tokens(
         self, entity_id: str
     ) -> tuple[collections.deque, ProtectApiClient] | None:
 
-        for entry in self.data.values():
-            if entity_id in entry.protect_data.access_tokens:
-                return entry.protect_data.access_tokens[entity_id], entry.protect
+        entries: list[ProtectData] = list(self.data.values())
+        for entry in entries:
```

Comment:
Linear search probably doesn't matter here

Reply:
Is there a better way to handle this? I do not think we can have a data store that persist outside of an entry, so iterating entries seems to be the only way to handle this.

Reply:
We'd have to make a `dict` index but its not worth it.